chat.go

```diff
@@ -435,6 +545,50 @@ func MsgOptionIconEmoji(iconEmoji string) MsgOption {
 	}
 }
 
+// MsgOptionCursor sets the cursor value when searching foor scheduled messages
```

Comment:
s/foor/for/

Reply:
My keyboard has seen better days.. this has been fixed.